rust/rbac-registration/src/cardano/cip509/cip509.rs

```diff
@@ -259,7 +310,7 @@ impl Cip509 {
 
     /// Returns URIs contained in both x509 and c509 certificates of `Cip509` metadata.
     #[must_use]
-    pub fn certificate_uris(&self) -> Option<&Cip0134UriSet> {
+    pub(crate) fn certificate_uris(&self) -> Option<&Cip0134UriSet> {
```

Comment:
I see that we need this function in Hermes ([here](https://github.com/input-output-hk/hermes/blob/main/hermes/apps/athena/modules/rbac-registration-indexer/src/lib.rs#L176-L179)).

Should `Hermes` handle this in a different way now after the refactor?

cc: @bkioshn